Python2 tests fail on Fedora 23

Running `make test PYTHON=python2` gives me
```
python2 -m pylint.lint --rcfile .pylintrc kafkajournalpump/
************* Module kafkajournalpump.daemon
C:199, 0: Unnecessary parens after 'print' keyword (superfluous-parens)
E: 32,15: Undefined variable 'FileNotFoundError' (undefined-variable)
E: 85,11: Undefined variable 'FileNotFoundError' (undefined-variable)
C: 96, 0: Old-style class defined. (old-style-class)
E:152,15: Undefined variable 'FileNotFoundError' (undefined-variable)
E:184,32: Undefined variable 'ProcessLookupError' (undefined-variable)
************* Module kafkajournalpump.kafkajournalpump
W:  9, 0: Module import itself (import-self)
C:205, 0: Old-style class defined. (old-style-class)
E:213,37: Module 'time' has no 'monotonic' member (no-member)
E:229,37: Module 'time' has no 'monotonic' member (no-member)
E:235,41: Module 'time' has no 'monotonic' member (no-member)
E:350,23: Module 'time' has no 'monotonic' member (no-member)
E:354,64: Module 'time' has no 'monotonic' member (no-member)
Makefile:15: recipe for target 'pylint' failed
make: *** [pylint] Error 22
```

OS: Fedora 23
python2 --version: 2.7.10

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,6 @@
 *~
 *.pyc
-/rpm
+/build/
+/dist/
+/kafkajournalpump.egg-info/
+/rpm/

--- a/.travis.yml
+++ b/.travis.yml
@@ -7,6 +7,7 @@ before_install:
 
 python:
   - "3.4"
+  - "3.5"
 
 install:
   - "pip install pylint pytest mock pep8 kafka-python coverage coveralls"

--- a/README.rst
+++ b/README.rst
@@ -32,6 +32,8 @@ Python/Other::
 
 This will produce an egg file into a dist directory within the same folder.
 
+kafkajournalpump requires Python 3.4 or newer.
+
 Installation
 ============
 
@@ -55,7 +57,7 @@ and eventually after the setup section, you can just run::
 
 Python/Other::
 
-  easy_install dist/kafkajournalpump-1.0.0-py2.7.egg
+  easy_install dist/kafkajournalpump-1.0.1-py3.4.egg
 
 On Debian/Other systems it is recommended that you run kafkajournalpump within
 a supervisord (http://supervisord.org) Process control system.

--- a/debian/copyright
+++ b/debian/copyright
@@ -1,6 +1,6 @@
 Format: http://dep.debian.net/deps/dep5
 Upstream-Name: kafkajournalpump
-Source: https://github.com/ohmu/kafkajournalpump/
+Source: https://github.com/aiven/kafkajournalpump/
 
 Files: *
 Copyright: 2015, Aiven <opensource@aiven.io>

--- a/kafkajournalpump.spec
+++ b/kafkajournalpump.spec
@@ -1,7 +1,7 @@
 Name:           kafkajournalpump
 Version:        %{major_version}
 Release:        %{minor_version}%{?dist}
-Url:            http://github.com/ohmu/kafkajournalpump
+Url:            http://github.com/aiven/kafkajournalpump
 Summary:        Pump messages from systemd journal to Kafka
 License:        ASL 2.0
 Source0:        kafkajournalpump-rpm-src.tar.gz

--- a/kafkajournalpump.spec
+++ b/kafkajournalpump.spec
@@ -1,9 +1,3 @@
-%if %{?python3_sitelib:1}0
-%global use_python3 1
-%else
-%global use_python3 0
-%endif
-
 Name:           kafkajournalpump
 Version:        %{major_version}
 Release:        %{minor_version}%{?dist}
@@ -11,14 +5,9 @@ Url:            http://github.com/ohmu/kafkajournalpump
 Summary:        Pump messages from systemd journal to Kafka
 License:        ASL 2.0
 Source0:        kafkajournalpump-rpm-src.tar.gz
-%if %{use_python3}
 Requires:       python3-kafka, systemd-python3
-BuildRequires:  python3-pytest, python3-pylint
-%else
-Requires:       python-kafka, systemd-python
-BuildRequires:  pytest, pylint
-%endif
 BuildRequires:  %{requires}
+BuildRequires:  python3-devel, python3-pytest, python3-pylint
 BuildArch:      noarch
 
 %description
@@ -34,22 +23,14 @@ kafkajournalpump is a daemon to pump journald messages into a given kafka topic.
 
 
 %install
-%if %{use_python3}
 python3 setup.py install --prefix=%{_prefix} --root=%{buildroot}
-%else
-python2 setup.py install --prefix=%{_prefix} --root=%{buildroot}
-%endif
 sed -e "s@#!/bin/python@#!%{_bindir}/python@" -i %{buildroot}%{_bindir}/*
 %{__install} -Dm0644 kafkajournalpump.unit %{buildroot}%{_unitdir}/kafkajournalpump.service
 %{__mkdir_p} %{buildroot}%{_localstatedir}/lib/kafkajournalpump
 
 
 %check
-%if %{use_python3}
-make test PYTHON=python3
-%else
-make test PYTHON=python2
-%endif
+make test
 
 
 %files
@@ -58,14 +39,13 @@ make test PYTHON=python2
 %{_bindir}/kafkajournalpump*
 %{_unitdir}/kafkajournalpump.service
 %{_localstatedir}/lib/kafkajournalpump
-%if %{use_python3}
 %{python3_sitelib}/*
-%else
-%{python_sitelib}/*
-%endif
 
 
 %changelog
+* Tue Feb 16 2016 Oskari Saarenmaa <os@aiven.io> - 1.0.1
+- We're Python 3 only now
+
 * Mon Aug 03 2015 Hannu Valtonen <hannu.valtonen@ohmu.fi> - 1.0.1
 - bugfix release
 

--- a/setup.py
+++ b/setup.py
@@ -11,5 +11,17 @@ setup(
     dependency_links=[],
     package_data={},
     data_files=[],
-    entry_points={'console_scripts': ["kafkajournalpump = kafkajournalpump.__main__:main"]}
+    entry_points={
+        "console_scripts": [
+            "kafkajournalpump = kafkajournalpump.__main__:main",
+        ],
+    },
+    classifiers=[
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Developers",
+        "Topic :: System :: Logging",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+    ],
 )

